passes/techmap/abc9.cc

```diff
@@ -952,6 +952,42 @@ struct Abc9Pass : public Pass {
 #endif
 #endif
 
+		// get arguments from scratchpad first, then override by command arguments
+		std::string lut_arg, luts_arg;
```

Comment:
By the way, some of the `string` declarations in this file use `std::` and some don't. Is there a preferred usage? I believe it's `yosys.h` that imports it...

Reply:
There's no official guidance as far as I know, it's a bit of a inconsistent mess right now....